webapp/src/websocket_handlers.ts

```diff
@@ -92,25 +92,21 @@ import {
 
 // NOTE: it's important this function is kept synchronous in order to guarantee the order of
 // state mutating operations.
-export function handleCallEnd(store: Store, ev: WebSocketMessage<EmptyData>) {
+export async function handleCallEnd(store: Store, ev: WebSocketMessage<EmptyData>) {
```

Comment:
Was the comment unclear, or am I missing something? :stuck_out_tongue: 
Either way, something has to go.

Reply:
I was playing around with a different refactor and that snuck through.